### PR TITLE
Use plain StandardJS to lint the code ⌨️

### PR DESCRIPTION
**Pull request by pirelenito**
- Remove all ESLint configurations from the project
- Remove `test:lint` script since linting is run through build and test

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,13 +18,26 @@
   "scripts": {
     "build": "rm -rf lib && babel src --out-dir lib --ignore *.spec.js",
     "test": "npm run test:lint && npm run test:unit && npm run test:integration",
-    "test:lint": "eslint .",
+    "test:lint": "standard | snazzy",
+    "test:lint:fix": "standard --fix",
```
Comment (npejo) [inline package.json]: shouldn't these new `scripts` be added to the README.md? Or they are supposed to be used only as part of `test` internally

File: src/configure-webpack/loaders/javascript.js [hunk 2]
```diff
@@ -83,10 +83,16 @@ export default {
...
+            loader: 'standard-loader',
             exclude: /node_modules/,
             options: {
-              configFile: path.join(projectPath, '.eslintrc')
+              env: {
+                jasmine: true
+              },
+              error: action !== actions.DEVELOP && action !== actions.TEST_UNIT,
+              parser: 'babel-eslint',
+              plugins: ['flowtype'],
```
Comment (npejo) [inline src/configure-webpack/loaders/javascript.js]: I can't find if the `standard-loader` supports `plugins` config key in the source code https://github.com/timoxley/standard-loader. Are you sure that we should have this here?

File: package.json (hunk 1 above)
Comment (pirelenito) [inline package.json]: They are internal scripts for the Sagui project itself, and therefore are not exposed to a project using Sagui.

File: src/configure-webpack/loaders/javascript.js (hunk 2 above)
Comment (pirelenito) [inline src/configure-webpack/loaders/javascript.js]: You can check https://github.com/timoxley/standard-loader/blob/master/index.js#L17

It is passed down to configure StandardJS.

**Pull request merged**